sphinx/cmd/make_mode.py

```diff
@@ -86,8 +86,8 @@ def build_clean(self) -> int:
             print("Error: '%s' directory contains source directory!" % self.build_dir)
             return 1
         print("Removing everything under '%s'..." % self.build_dir)
-        for item in self.build_dir.iterdir():
```

Comment:
On reflection, I'm not sure whether the change on this line (using the pre-resolved `build_dir`) is worthwhile.  Perhaps it should be applied in a separate pull request, if at all.